Code/GraphMol/MolEnumerator/enumerator_catch.cpp

```diff
@@ -1661,4 +1661,37 @@ TEST_CASE("MolEnumerator should propagate atom properties") {
   CHECK(bundle[0]->getAtomWithIdx(7)->hasProp("_foo"));
   CHECK(bundle[1]->getAtomWithIdx(8)->hasProp("_foo"));
   CHECK(bundle[2]->getAtomWithIdx(9)->hasProp("_foo"));
-}
\ No newline at end of file
+}
+
+TEST_CASE("SRU enumeration should adhere to repeat counts #6429") {
```

Comment:
This test looks good, but in the name of "trust, but verify" I think it would be good to also have something equivalent directly from V3000 mol blocks.

Reply:
good point! thanks